qcodes/instrument/parameter.py

```diff
@@ -1295,6 +1296,18 @@ def __init__(
                 '',
                 self.__doc__))
 
+    @property
+    def unit(self) -> str:
```

Comment:
needs a news fragment

Reply:
I honestly dont think so. From the end users point of view there is zero difference. But sure we can add it

Reply:
ah, you're right. well, there is a difference for those implementing subclasses of Parameter :)

Reply:
added one anyway